File: app/src/main/java/com/caramelheaven/lennach/models/model/common/Captcha.java
package com.caramelheaven.lennach.models.model.common;

public class Captcha {
    private String id;
    private Integer result;
    private String type;

    public Integer getResult() {
        return result;
    }

    public void setResult(Integer result) {
        this.result = result;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/com/caramelheaven/lennach/datasource/model/Captcha.kt
package com.caramelheaven.lennach.datasource.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Captcha {
    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("result")
    @Expose
    var result: Int? = null
    @SerializedName("type")
    @Expose
    var type: String? = null
}
